File: NoLookKeyboard_java_eclipse/src/com/teuskim/nlkeyboard/GuideActivity.java
package com.teuskim.nlkeyboard;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;

public class GuideActivity extends Activity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.guide);
		NLPreference pref = new NLPreference(getApplicationContext());
		pref.setDidReadGuide(true);
		
		findViewById(R.id.btn_close).setOnClickListener(new View.OnClickListener() {
			
			@Override
			public void onClick(View v) {
				finish();
			}
		});
	}

}

File: NoLookKeyboard_kotlin/app/src/main/java/com/teuskim/nlkeyboard/GuideActivity.kt
package com.teuskim.nlkeyboard

import android.app.Activity
import android.os.Bundle
import android.view.View

class GuideActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.guide)
        val pref = NLPreference(applicationContext)
        pref.setDidReadGuide(true)

        findViewById<View>(R.id.btn_close).setOnClickListener { finish() }
    }

}
